src/components/v1/shared/V1ProjectHandle.tsx

```diff
@@ -14,24 +13,21 @@ export default function V1ProjectHandle({
   style?: CSSProperties
   link?: boolean
 }) {
+  const [hover, setHover] = useState(false)
```

Comment:
**Suggestion:** think we can avoid user hover states by using the class `hover-action`

Reply:
alright I'm on it